src/admin-host-server.ts

```diff
@@ -17,6 +18,20 @@ export default (port, baseDir: string, masterClient) => {
       .catch(catchHttp(next))
   })
 
+  app.post('/holo/happs/enable', async (req, res, next) => {
+    const {happId}: InstallHappRequest = req.body
+    HH.enableHapp(masterClient, happId)
+      .then(() => res.send("Installation successful"))
```

Comment:
Could we change this to Enabled Succesfully and Disabled Succesfully respectively.

Reply:
Good catch, updated in develop